### PR TITLE
Rgo2019

### LINKED ISSUE #794: RGO2019 PR hero_description

### PR DESCRIPTION
**Pull request by KevinDang74**
New poses for take out the garbage challenge

fixes https://github.com/tue-robotics/tue_robocup/issues/794

### CODE REVIEW

File: custom/skills.yaml [hunk 1]
```diff
@@ -24,7 +26,11 @@ skills:
             carrying_bag_pose: [0.01, 0.0, -1.57, -1.57, 0.0]
             driving_bag_pose:  [0.01, 0.0, -1.57, -1.57, 0.0]
             drop_bag_pose:     [0.01, 0.0, -1.57, -1.57, 0.0]
+            point_at:          [0.40, -0.4, 0,    -1.17, 1.57]
             zero:              [0.01, 0.0, -1.57, -1.57, 0.0]
+            arm_out_of_way:   [0.01, -0.3, -1.57, -1.57, 0.0]
```
Comment (MatthijsBurgh) [inline custom/skills.yaml]: Please fix indentation alignment

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: custom/skills.yaml (hunk 1 above)
Comment (KevinDang74) [reply custom/skills.yaml]: Fixed the indentation

File: custom/skills.yaml [hunk 2]
```diff
@@ -24,7 +26,11 @@ skills:
             carrying_bag_pose: [0.01, 0.0, -1.57, -1.57, 0.0]
             driving_bag_pose:  [0.01, 0.0, -1.57, -1.57, 0.0]
             drop_bag_pose:     [0.01, 0.0, -1.57, -1.57, 0.0]
+            point_at:          [0.40, -0.4, 0,    -1.17, 1.57]
```
Comment (JosjaG) [inline custom/skills.yaml]: For consistency: use 0.0 and remove redundant spaces

Comment (JosjaG) [reply custom/skills.yaml]: Updated this

**Review by MatthijsBurgh — APPROVED** [review]

**Pull request merged**